### PR TITLE
Create signup page

### PR DESCRIPTION
**Pull request by AyaAljalees**

### CODE REVIEW

**Review by Amal-Mousa — APPROVED** [review]

**Review by KhaledToha — APPROVED** [review]

**Review by Mohammed-Sbeata — APPROVED** [review]
Good Job👏👏👏

File: src/components/Signup/index.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        <SignupForm />
+      </CustomContainer>
+      <DontHaveAcount>
+        Already have an account ?
+      </DontHaveAcount>
+      <SigninText>
+        Sign in
+      </SigninText>
+    </BodyContainer>
+    <SignupImage src="src/assets/todo.svg" alt="login" />
```
Comment (nada-abuzaid) [inline src/components/Signup/index.tsx:36]: import the image and then use it

File: src/components/Signup/signup.styled.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  borderRadius: '4.125rem',
+  borderColor: custom.white,
+}));
+
+export const DontHaveAcount = styled(Typography)(() => ({
+  position: 'absolute',
+  top: '44.375rem',
+  fontSize: '0.6875rem',
+  right: '10.625rem',
+  fontFamily: 'Montserrat',
```
Comment (nada-abuzaid) [inline src/components/Signup/signup.styled.tsx:62]:
```suggestion
  fontFamily: 'Poppins',
```

File: src/components/Signup/SignupForm/signupForm.tsx [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+  textTransform: 'none',
+}));
+
+export const GooglePlusButton = styled(LoadingButton)(() => ({
+  position: 'absolute',
+  width: '23.75rem',
+  height: '3.375rem',
+  right: '0.76875rem',
+  top: '40rem',
+  fontFamily: 'Montserrat',
```
Comment (nada-abuzaid) [inline src/components/Signup/SignupForm/signupForm.tsx:159]:
```suggestion
  fontFamily: 'Poppins',
```

**Review by nada-abuzaid — APPROVED** [review]

**Pull request merged**